torch/linalg/__init__.py

```diff
@@ -633,38 +679,39 @@
 The drivers `'gelsy'`, `'gelsd'`, `'gelss'` handle rank-deficient inputs.
 See the note below on how to choose the best driver. See also the `full description of these drivers`_
 
-:attr:`cond` is used to determine the effective rank of the matrices in :attr:`input`
+:attr:`cond` is used to determine the effective rank of the matrices in :attr:`A`
 for rank-revealing drivers, that is, when :attr:`driver` is one of (`'gelsy'`, `'gelsd'`, `'gelss'`).
 In this case, if :math:`\sigma_i` are the singular values of `A` in decreasing order,
 :math:`\sigma_i` will be rounded down to zero if :math:`\sigma_i \leq \text{cond} \cdot \sigma_1`.
-If :attr:`cond` `= None` (default), :attr:`cond` is set to the machine precision of the dtype of :attr:`input`.
+If :attr:`cond` `= None` (default), :attr:`cond` is set to the machine precision of the dtype of :attr:`A`.
 
-This function returns the solution to the problem and some extra information in a namedtuple of
+This function returns the solution to the problem and some extra information in a named tuple of
```

Comment:
Consider moving this to a returns section

Reply:
This one is too big, and the return too complex. I'm leaving it here, but I'm also adding a `Returns` section with the names of the named tuple.